index.html

```diff
@@ -68,15 +68,22 @@ <h2>
       TBW
     </p>
     <p id="credential-type-examples">
-      There are many types of digital credential that can be conveyed using
+      There are many types of digital credential that can be presented using
       this API. <dfn data-lt="credential type examples">Examples of these
       types</dfn> include:
     </p>
     <ul>
-      <li>a driver's license or identity card issued by a government
+      <li>a driving license, passport, or other identity card issued by a government
       institution
       </li>
-      <li>TBW: other credential types
+      <li>a travel authorization document issued by an embassy or consulate
+      </li>
+      <li>a proof of employment issued by a private organization
+      </li>
+      <li>a proof of education or professional training issued by an institution
+      </li>
+      <li>and many other scenarios as described in 
+        <a href="https://www.w3.org/TR/vc-use-cases/">Verifiable Credentials Use Cases</a>
```

Comment:
These are great use cases. It is not perfectly clear to me whether all of these are going to be best served by this API, but I think is a great baseline to work from.